### PR TITLE
added tasks for shortener database and infra-web, added secrets for d…

### PR DESCRIPTION
**Pull request by gal**
…b for shortener

### CODE REVIEW

File: provision-infra-web.yml [hunk 1]
```diff
@@ -1405,4 +1405,87 @@
...
+                DB_HOST: "{{ interfaces.infra.databases.net0.addresses[0] | ipaddr('address') }}"
+                DB_PORT: "{{ shortener.postgres.port | default(5432) }}"
+                DB_USER: "{{ shortener.postgres.username }}"
+                DB_PASSWORD: "{{ shortener.postgres.password }}"
+                DB_DATABASE: "{{ shortener.postgres.db }}"
+              networks:
+                traefik:
+              labels:
+                traefik.enable: "true"
+                traefik.http.routers.shortener.rule: "Host(`links.netsoc.co`)"
```
Comment (ericm) [inline provision-infra-web.yml:1469]: Is this set in the `setup-external-dns`?

Comment (gal) [reply provision-infra-web.yml:1469]: it is now :wink:

File: provision-infra-web.yml [hunk 2]
```diff
@@ -1405,4 +1405,87 @@
     - name: LOKI TARGETS - Setup targets
       shell: "docker ps --format '- targets: [\"{{ '{{' }}.ID{{ '}}' }}\"]\\n  labels:\\n    container_name: \"{{ '{{' }}.Names{{ '}}' }}\"' | sed -e '/`docker ps -aqf \"name=loki_promtail_1\"`/,+3d'  | sed -e '/`docker ps -aqf \"name=loki_loki_1\"`/,+3d' > /netsoc/promtail/targets.yaml"
   tags:
-    - always
\ No newline at end of file
+    - always
+
+- name: Shortener Databases
```
Comment (ocanty) [inline provision-infra-web.yml:1410]: needs to be before loki targets

Comment (gal) [reply provision-infra-web.yml:1402]: sorry for a rtfm moment

**Review by ericm — APPROVED** [review]
Try running it if you havent and merge if it works

**Comment by gal** [comment]
Working now

File: provision-infra-web.yml [hunk 3]
```diff
@@ -1398,11 +1401,102 @@
...
+                DB_HOST: "{{ interfaces.infra.databases.net0.addresses[0] | ipaddr('address') }}"
+                DB_PORT: "5432"
+                DB_USER: "{{ shortener.postgres.user }}"
+                DB_PASS: "{{ shortener.postgres.pass }}"
+                DB_NAME: "{{ shortener.postgres.db }}"
+              networks:
+                traefik:
+              labels:
+                traefik.enable: "true"
+                traefik.http.routers.shortener.rule: "Host(`{{ shortener.traefik.domain }}`)"
```
Comment (ericm) [inline provision-infra-web.yml:1463]: Do we need the domain as a secret?

Comment (gal) [reply provision-infra-web.yml:1463]: do you want me to keep it as a variable, or just hardcode it in?

Comment (ericm) [reply provision-infra-web.yml:1463]: Hardcoded is grand :p

Comment (gal) [reply provision-infra-web.yml:1463]: its used in the bot's environment variables too, so i thought it might be appropriate

Comment (ericm) [reply provision-infra-web.yml]: I'd say just keep it inline, makes it more readable :)

**Comment by ericm** [comment]
FANTASTIC

**Comment by gal** [comment]
> FANTASTIC

:CRINGE:

**Review by ericm — APPROVED** [review]

**Pull request merged**